components/compass_theme_provider/compass_theme_provider.tsx

```diff
@@ -20,6 +87,135 @@ const CompassThemeProvider = ({theme, children}: Props): JSX.Element | null => {
         noFontFaces: true,
     });
 
+    //     if (!UserAgent.isFirefox() && !UserAgent.isInternetExplorer() && !UserAgent.isEdge()) {
+    //         changeCss('body.app__body ::-webkit-scrollbar-thumb', 'background:' + changeOpacity(theme.centerChannelColor, 0.4));
+    //     }
+
+    const CssVariables = createGlobalStyle`
+        :root {
```

Comment:
0/5, but perhaps we might want to consider only applying the CSS variables to the children of this component rather than at the root of the app. That would allow us to nest theme providers in the future if we ever wanted to theme one part of the app differently from others. For example, in a hackathon demo Devin and I did way back, we added a preview of a user's theme in the custom theme settings, and that was coloured using a nested ThemeProvider component

Reply:
that would simplify things ... we can still change that when we need to ... its pretty much up to how you want to handle that in the future